docs/html/dropdown/dropdown--optgroups.html

```diff
@@ -0,0 +1,23 @@
+<div className="ray-dropdown">
```

Comment:
These .html files need to be valid html. These look like JSX, make sure you convert things like `className` to `class`, and `htmlFor` to `for`

Reply:
thanks. copy-paste plays bad role

Reply:
done